docs/components/board/pi5.md

```diff
@@ -0,0 +1,167 @@
+      "pin": "<pin-number-on-adc>",
+      "spi_bus": "<your-spi-bus-name>",
+      "average_over_ms": <int>,
+      "samples_per_sec": <int>
+    }
+  ],
+  "digital_interrupts": [
+    {
+      "name": "<your-digital-interrupt-name>",
+      "pin": "<pin-number>",
+      "type": "<basic|servo>"
+    }
+  ],
+  "spis": [
+    {
+      "name": "<your-bus-name>",
+      "bus_select": "<your-bus-index>"
+    }
+  ],
+  "i2cs": [
+    {
+      "name": "<your-bus-name>",
+      "bus": "<your-bus-index>"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="Attributes example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "digital_interrupts": [
+    {
+      "name": "your-interrupt-1",
+      "pin": "15"
+    },
+    {
+      "name": "your-interrupt-2",
+      "pin": "16"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-pi-board-name>",
+      "model": "pi5",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "analogs": [
+          <...See table below...>
+        ],
+        "digital_interrupts": [
+          <...See table below...>
+        ],
+        "spis": [
+          <...See table below...>
+        ],
+        "i2cs": [
+          <...See table below...>
+        ]
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `pi5` boards:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `analogs` | object | Optional | Attributes of any pins that can be used as analog-to-digital converter (ADC) inputs. See [configuration info](#analogs). |
+| `digital_interrupts` | object | Optional | Any digital interrupts's {{< glossary_tooltip term_id="pin-number" text="pin number" >}} and name. See [configuration info](#digital_interrupts). |
+| `spis` | object | Optional | Any Serial Peripheral Interface (SPI) chip select pins' bus index and name. See [configuration info](#spis). Review [these instructions](/get-started/installation/prepare/rpi-setup/#enable-communication-protocols) to learn how to enable SPI on a Raspberry Pi. |
+| `i2cs` | object | Optional | Any Inter-Integrated Circuit (I<sup>2</sup>C) pins' bus index and name. See [configuration info](#i2cs). Review [these instructions](/get-started/installation/prepare/rpi-setup/#enable-communication-protocols) to learn how to enable I<sup>2</sup>C on a Raspberry Pi. |
+
+## Attribute Configuration
+
+Configuring these attributes on your board allows you to integrate [analog-to-digital converters](#analogs), [digital interrupts](#digital_interrupts), and components that must communicate through the [SPI](#spis) and [I<sup>2</sup>C](#i2cs) protocols into your robot.
+
+### `analogs`
+
+{{< readfile "/static/include/components/board/board-analogs.md" >}}
```

Comment:
I get that this part is standardized across all our boards. However, the thing that we've standardized on is plugging an MCP300x chip into the SPI bus and using that, and we don't want to mislead people into thinking their board has a built-in analog reader (the way the Raspberry Pi Pico does) or thinking they can plug in a different ADC chip. I wish we would split the ADC stuff out to a separate component, rather than being in the board. 

Is it worth rephrasing this? Part of me says yes, because at the moment I think it's confusing. Part of me says not in this PR and possibly not even in its own PR, because it's off-topic for this PR and the rest of the company doesn't seem to get confused by it.

Reply:
Mm yeah I think that's outside the scope here since it's currently not considered a separate component right? And we can remove this here and also from the other boards in a different PR if and when it becomes the thing to do.